proto/validator/genesis.proto

```diff
@@ -16,5 +17,6 @@ message GenesisState {
   repeated LastValidatorPower lastValidatorPowerList = 2 [(gogoproto.nullable) = false];
   repeated ProposedDisableValidator proposedDisableValidatorList = 3 [(gogoproto.nullable) = false];
   repeated DisabledValidator disabledValidatorList = 4 [(gogoproto.nullable) = false];
+  repeated RejectedDisableNode rejectedNodeList = 5 [(gogoproto.nullable) = false];
```

Comment:
There is inconsistency between `ProposedDisableValidator` and `RejectedDisableNode`.
Consider renaming `RejectedDisableNode` to `RejectedDisableValidator`.